common/constants.go

```diff
@@ -303,3 +303,9 @@ const (
 	DeviceSystemEventActionUpdate = "update"
 	DeviceSystemEventActionDelete = "delete"
 )
+
+const (
+	ConfigStemApp    = "edgex/appservices/"
+	ConfigStemCore   = "edgex/core/"
+	ConfigStemDevice = "edgex/devices/"
+)
```

Comment:
Missing `ConfigStemSecurity = "edgex/security/"`

Reply:
I'm not sure its needed.  I checked the original PR that we are trying to refactor and `ConfigStemSecurity ` was unused.

Reply:
It was missed on the original PR and all need to be moved for consistency
https://github.com/edgexfoundry/edgex-go/blob/main/internal/constants.go#L22-L25

Reply:
Agreed.  Checked latest code and this is true.